### PR TITLE
PREAPPS-4347 :  Reskin of error.jsp and remove of url from 2fa and forgetpassword flow

### PR DESCRIPTION
**Pull request by auminpatel**
reskin of error.jsp page and removing zimbra url from 2fa and forgort-password flow and url for the login page can be cusotomised in WebRoot/skins/harmony/skin.properties

### CODE REVIEW

File: WebRoot/css/login.css [hunk 1]
```diff
@@ -520,4 +520,41 @@ label#remember {
...
+}
+
+.errorBox h2 {
+	margin: 0 0 24px;
+	font-size: 20px;
+	font-weight: normal;
+}
+
+.errorBox p {
+	font-size: 13px;
```
Comment (silentsakky) [inline WebRoot/css/login.css]: aren't we using legacy defined font size variables?

File: WebRoot/public/error.jsp [hunk 2]
```diff
@@ -29,14 +29,13 @@
 
 <%
         String skin = request.getParameter("skin");
-        boolean skinOK = (skin != null &&
-                          Pattern.matches("^[0-9A-Za-z]+$", skin));
+        if (skin == null || !Pattern.matches("^[0-9A-Za-z]+$", skin)){
+			skin = application.getInitParameter("zimbraDefaultSkin");
+		}
```
Comment (silentsakky) [inline WebRoot/public/error.jsp]: indentation

File: WebRoot/public/error.jsp [hunk 3]
```diff
@@ -69,37 +69,17 @@
...
-					</table>
-				</div>
-			</div>
-			<div class="decor1"></div>
+		<div class="errorBox">
+			<h2><fmt:message key="${errTitle}"/></h2>
+			<p>
+				<fmt:message key="${errMsg}"/><br/>
+				<fmt:message key="errorTryAgainLater"/><br/><br/>
+				Error ${errCode}
```
Comment (silentsakky) [inline WebRoot/public/error.jsp]: translations?

File: WebRoot/templates/share/Dialogs.template [hunk 4]
```diff
@@ -725,9 +725,7 @@
 <template id='share.Dialogs#ZmTwoFactorCustomLoginPage'>
 	<div class="ZmTwoFactorCustomSetupLoginPage">		
 		<div class="logo">
-			<a href="https://www.zimbra.com/"  style=" outline :none;" id="bannerLink" target="_new" title='<fmt:message key="zimbraTitle"/>'>
-				<img style="outline:none;" src="/img/new-logo.png" width="130" height="32" border="0" alt="Zimbra" title="Zimbra">
-			</a>
+			<img style="outline:none;" src="/img/new-logo.png" width="130" height="32" border="0" alt="Zimbra" title="Zimbra">
```
Comment (silentsakky) [inline WebRoot/templates/share/Dialogs.template]: why have we removed links to zimbra.com ?

File: WebRoot/public/error.jsp (hunk 3 above)
Comment (auminpatel) [reply WebRoot/public/error.jsp]: translations are already present , haven’t chnaged the keys have just reskined it

Comment (silentsakky) [reply WebRoot/public/error.jsp]: `Error ${errCode}` is not translated

Comment (auminpatel) [reply WebRoot/public/error.jsp]: it is present in same way here in legacy code so i dint add 
<p style="margin-bottom:2em;">ERROR: ${errCode}</p>

File: WebRoot/templates/share/Dialogs.template (hunk 4 above)
Comment (auminpatel) [reply WebRoot/templates/share/Dialogs.template]: earlier we dint have , urls here and we only had logo , so had talk with shardool and he suggested to remove it from both

File: WebRoot/css/login.css (hunk 1 above)
Comment (auminpatel) [reply WebRoot/css/login.css]: have used a variable whose font-size is almost matching 13px

File: WebRoot/public/error.jsp (hunk 2 above)
Comment (auminpatel) [reply WebRoot/public/error.jsp]: Done

**Pull request merged**